documentation/static/test-and-development/did.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:web:uncefact.github.io:project-vckit:test-and-development",
+  "verificationMethod": [
```

Comment:
To verify Enveloping Proof, we need to type `JsonWebKey` following W3C document [here](https://www.w3.org/TR/vc-jose-cose/#using-controller-documents). Do you think we should update it?
<img width="1141" alt="image" src="https://github.com/user-attachments/assets/146937b2-cec8-477a-ad06-7c70c69c168c">